tests/gradient_descent_test.py

```diff
@@ -122,6 +124,18 @@ def solve(hyperparams):
     error0 = jax.jit(solve)(hyperparams_list[0])
     self.assertAllClose(errors[0], error0)
 
+  def test_n_calls(self):
```

Comment:
please add a short docstring to this function

Reply:
done